MODS.md

```diff
@@ -181,3 +181,5 @@ You can specify the following options:
 
 - `--output=<file>`: Write the output to a file instead of standard output.
 - `--codegen`: Generates an approximatation of the GDScript source code.
+
+Or use this pre-decompiled version of the game: [Game Source](https://github.com/ai-kana/WebfishingDatamining)
```

Comment:
Hosting decompiled copies of a game on GitHub is like super ultra turbo illegal. Please remove this with a force push, I'm not encouraging this.

Reply:
Done